src/slicing.jl

```diff
@@ -56,24 +56,24 @@ function roi(inds::Indices, dims::Dims{2})
             push!(axs, Axis{i}(ind))
         end
     end
-    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...), (axs...))
+    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...,), (axs...,))
 end
 
-function roi(axs::NTuple{N,Axis}, axes::Tuple{Symbol,Symbol}) where N
-    axes[1] != axes[2] || error("entries in axes must be distinct, got ", axes)
+function roi(axs::NTuple{N,Axis}, axnames::Tuple{Symbol,Symbol}) where N
+    axnames[1] != axnames[2] || error("entries in axnames must be distinct, got ", axnames)
     names = axisnames(axs...)
-    dims = indexin([axes...], [names...])
-    inds = map(v->indices(v, 1), axisvalues(axs...))
+    dims = indexin([axnames...], [names...])
+    inds = map(v->axes(v, 1), axisvalues(axs...))
     zr = ZoomRegion(inds[[dims...]])
     sigs, axs = [], []
     for (i, n) in enumerate(names)
-        if !(n ∈ axes)
+        if !(n ∈ axnames)
             ind = inds[i]
             push!(sigs, Signal(first(ind)))
             push!(axs, Axis{n}(ind))
         end
     end
-    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...), (axs...))
+    Signal(zr), SliceData{dims[2] < dims[1]}((sigs...,), (axs...,))
```

Comment:
is the trailing `,` here needed now?

Reply:
I think so, and in fact I believe this was added by Femtocleaner (which I ran locally).
```julia
julia> x = (1,2)
(1, 2)

julia> s = (x...)
┌ Warning: Deprecated syntax `(x...)` at REPL[2]:1.
│ Use `(x...,)` instead.
└ @ REPL[2]:1
(1, 2)
```